Helpers.ps1

```diff
@@ -815,22 +821,23 @@ function Repair-LogFile
     .EXAMPLE
         Repair-LogFile
 #>
-    $path = (Get-GitHubConfiguration -Name LogPath)
-
-    $logFileBytes = Get-Content -Encoding Byte -ReadCount 2 -Path $path
-
-    if ($logFileBytes -eq @(254, 255))
+    $path = (Get-GitHubConfiguration -Name LogPath -ErrorAction SilentlyContinue)
```

Comment:
`Get-GitHubConfiguration` won't throw, so you don't need the `-ErrorAction SilentlyContinue` on this.   However, you don't know if you're getting back a path that actually exists.  It may point to a totally legitimate path in the user's Documents directory, however if they disabled logging, then the file still won't exist.  Simply checking that `$path` is not null isn't sufficient.

You either have to use `Test-Path` to _check_ if the file exists, or you have to use `-ErrorAction SilentlyContinue` when you call `Get-Content`.